### PR TITLE
Create the dpdk namespace in the deploy overlay

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Comment by SchSeba** [comment]
@yuvalk PTAL

File: feature-configs/deploy/dpdk/build-config.yaml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dpdk
+---
```
Comment (yuvalk) [inline feature-configs/deploy/dpdk/build-config.yaml:6]: why not have it in a file of it's own?
it's harder to search for it that way
also impossible to remove just part, without editing the file

Comment (SchSeba) [reply feature-configs/deploy/dpdk/build-config.yaml]: done

**Pull request merged**